core/modules/installer/theme/installer.theme.inc

```diff
@@ -0,0 +1,224 @@
+  // Add the title and content variables.
+  $variables['title'] = $variables['element']['#title'];
+  $variables['content'] = $variables['element']['#content'];
+}
+
+/**
+ * Add some variables for the projects list theme.
+ *
+ * @param $variables
+ *   An associative array containing:
+ *     - projects_list : array of all projects.
+ */
+function update_preprocess_installer_browser_list(&$variables) {
+  module_load_include('inc', 'installer', 'installer.browser');
+  backdrop_add_css(backdrop_get_path('module', 'installer') . '/css/installer.css');
+
+  if (is_array($variables['projects_list']) && !empty($variables['projects_list'])) {
+    $content = '';
+    $first = TRUE;
+    $rows = array();
+    switch ($variables['type']) {
+      case 'module':
+        $title = t('Modules');
+        break;
+      case 'theme':
+        $title = t('Themes');
+        break;
+      case 'layout':
+        $title = t('Layouts');
+        break;
+      default:
+        $title = t('Projects');
+        break;
+    }
+    $headers = array(
+      $title,
+    );
+    // Theme each individual project and add to the list.
+    foreach ($variables['projects_list'] as $project) {
+      $row = array();
+      $row[] = theme('installer_browser_project', array('project' => $project, 'first' => NULL));
+      $rows[] = $row;
+    }
+    $content .= theme('table', array('header' => $headers, 'rows' => $rows));
+  }
+  else {
+    $content = t('No results found.');
+  }
+
+  $variables['main_content'] = render($content);
+
+  // Add the pager.
+  $variables['pager'] = theme('pager', array('tags' => NULL));
+
+  // Add the filters.
+  $filter_content = backdrop_get_form('installer_browser_filters_form', $variables['type']);
+  $filters['installer_browser_filters_block'] = array(
+    '#theme' => 'installer_browser_block',
+    '#title' => t('Filter search'),
+    '#content' => backdrop_render($filter_content),
+  );
+  $variables['filters'] = render($filters);
+
+  // Add the install list.
+  $install_list['installer_browser_filters_block'] = array(
+    '#theme' => 'installer_browser_block',
+    '#title' => t('Installation queue'),
+    '#content' => installer_browser_get_install_list(),
+  );
+  $variables['install_list'] = render($install_list);
+
+  // Add the advanced options block.
+  $advanced_content = backdrop_get_form('installer_browser_manual_install_link');
+  $variables['advanced'] = backdrop_render($advanced_content);
+}
+
+/**
+ * Add some variables for the project details dialog.
+ *
+ * @param $variables
+ *   An associative array containing:
+ *     - project : associative array of project variables.
+ */
+function update_preprocess_installer_browser_project_dialog(&$variables) {
+  update_preprocess_installer_browser_project($variables);
+  $project = $variables['project'];
+  $variables['project_page'] = l(t('View on BackdropCMS.org'), 'https://backdropcms.org/project/' . $project['name']);
```

Comment:
for some reason clicking on this link opens six new tabs for me. Does it do the same for anyone else?